### DISCUSSION TITLE
Paste image from clipboard and use base64 string instead of uploading too

### DISCUSSION THREAD

**Discussion by GuyoST** [react-page/react-page#1111] [Q&A]
Hello every one,

I just discovered React-Page and I spent time testing it and read the doc and it's amazing !
I just noticed that in a text cell (or even in a image cell) , it's not possible to add an image from the clipboard (copy / paste), the same way it's allowed in many editors.

I think that would be awesome to copy / paste an image in a text cell, with simple options : like right / left / center and vertical align also. Or to use the existing image cell : when we paste an image from the clipboard, it adds an image cell automatically and use a base64 src.

Maybe the image cell should allow to paste or import the image from the clipboard and add it as base64 string directly ?

Or maybe I missed something and it's already possible ?

Thank you very much :) !!

**Comment by macrozone** [comment]
currently you cant insert images with copy & paste.

We could do that though, but it would probably add an image cell. 

storing the image as a base64 string is probably not a good idea, as this can be huge.

**Reply by GuyoST** [reply]
Yes, I think adding image cell would be the best :) !
That would be great !!

Thank you so much :)
